pkg/check/datastore.go

```diff
@@ -201,7 +205,7 @@ func getPolicyDatastores(ctx *CheckContext, profileID types.PbmProfileId) ([]str
 	var content []vim.ObjectContent
 	tctx, cancel = context.WithTimeout(ctx.Context, *Timeout)
 	defer cancel()
-	err = v.Retrieve(tctx, kind, []string{"Name"}, &content)
+	err = v.Retrieve(tctx, kind, []string{"name"}, &content)
```

Comment:
Why is this change needed? Is there any risk it breaks datastore lookups where the key is `Name`?

Reply:
It appears that, using `Name` didn't work.  I don't have logs right now but it resulted in "Invalid property" errors and the whole check was getting skipped. 

Only with OCP-4.10 we started shipping storageclasses that use storage policies and there too we aren't checking CSI based Storageclass and hence this issue flew under the radar.  cc @jsafrane

Reply:
Actually I wrote a simple standalone program and following code:

```go
var content []vim.ObjectContent
	err = v.Retrieve(context.TODO(), kind, []string{"Name"}, &content)
	_ = v.Destroy(context.TODO())
	if err != nil {
		debug("getPolicyDatastores: error listing datastores: %v", err)
		return
	}
```

results in :

```
getPolicyDatastores: error listing datastores: ServerFaultCode: InvalidProperty
```

And changing it to `name` does not result in error. I don't know if it *ever* worked. :(

Reply:
Doh! Ok, thanks for the context.